syntax = "proto3";
import "cluster.proto";
import "base.proto";
package ProtoMsg;

//SVR_MSG_REGISTER_LOGIN
message PbSvrRegisterLoginReqMsg {
   PbSvrBaseInfo        svr_info = 1;
};

message PbSvrRegisterLoginResMsg {
   Ret ret = 1;
};
